tests/client/LoginTests/Login.spec.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { expect } from 'chai';
+// import Login from '../../../../src/client/components';
+
+describe('Component: <Login />', () => {
+  it('should have two input boxes and three buttons', () => {
+    const wrapper = shallow(<Login />);
+    expect(wrapper.find('input')).to.have.length(2);
+    expect(wrapper.find('button')).to.have.length(3);
+  });
+
+  it('should have props of loginUser, isPending, and isLoggedIn', () => {
```

Comment:
Good point -- just wanted to hash out the props in this testing spec, which can probably be scrubbed once we implement propTypes.